pkg/project/project.go

```diff
@@ -1,7 +1,7 @@
 package project
 
 var (
-	bundleVersion        = "8.0.0"
+	bundleVersion        = "8.1.0-dev"
```

Comment:
According to https://github.com/giantswarm/giantswarm/blob/master/areas/empowerment/versioning-of-operators/how-to-release-an-operator.md this should be `8.0.1-dev`. IMO this is weird to lower the version if we need to make a patch release. I'm not big on that though as this is supposed to be convenient for other teams. But if we want it that way please update the guide.

Reply:
With you. Thanks for notice. We are just getting warm with the process.